### PR TITLE
[ODS-6178] Updates for ODS/API v6.2

### PR DESCRIPTION
**Pull request by stephenfuqua**
Note that this includes additional improvements from:

* ODS-6061
* ODS-6054
* ODS-5778

### CODE REVIEW

File: Web-Ods-AdminApp/Alpine/pgsql/Dockerfile [hunk 1]
```diff
@@ -3,11 +3,17 @@
...
 
-# Tag aspnet:6.0-alpine
-FROM mcr.microsoft.com/dotnet/aspnet@sha256:201cedd60cb295b2ebea7184561a45c5c0ee337e37300ea0f25cff5a2c762538
+FROM mcr.microsoft.com/dotnet/aspnet:8.0.3-alpine3.19-amd64@sha256:a531d9d123928514405b9da9ff28a3aa81bd6f7d7d8cfb6207b66c007e7b3075 AS base
+RUN apk --no-cache add unzip=~6 dos2unix=~7 bash=~5 gettext=~0 icu=~74 curl=~8 && \
+    addgroup -S edfi && adduser -S edfi -G edfi
+
+# Start a new layer so that the above layer can be cached
+FROM base AS build
+RUN apk --no-cache postgresql13-client=~13 jq=~1
```
Comment (andonyns) [inline Web-Ods-AdminApp/Alpine/pgsql/Dockerfile:12]:
```suggestion
RUN apk add --no-cache postgresql13-client=~13 jq=~1
```

**Review by andonyns — CHANGES_REQUESTED** [review]

File: Web-Ods-AdminApp/Alpine/pgsql/Dockerfile (hunk 1 above)
Comment (andonyns) [reply Web-Ods-AdminApp/Alpine/pgsql/Dockerfile:12]: After this suggestion is applied, it's ready to merge

Comment (stephenfuqua) [reply Web-Ods-AdminApp/Alpine/pgsql/Dockerfile:12]: Did Admin App actually work? It is still programmed on .NET 6 or maybe even .NET 3.1 (I haven't looked), so I am curious what happens when we try to run it in a container that has .NET 8.

Comment (stephenfuqua) [reply Web-Ods-AdminApp/Alpine/pgsql/Dockerfile]: Done

**Review by andonyns — APPROVED** [review]

**Pull request merged**